bsp/boards/radio.h

```diff
@@ -66,7 +66,7 @@ void     radio_setFrequency(uint8_t frequency);
 void     radio_rfOn(void);
 void     radio_rfOff(void);
 // TX
-void     radio_loadPacket(uint8_t* packet, uint8_t len);
+void     radio_loadPacket(uint8_t* packet, uint16_t len);

```

Comment:
right, the .h file applies to all boards.  On the other hand, I didn't get any e-mail. I'll check the notification settings.